gci/gitorg.py

```diff
@@ -85,3 +87,14 @@ def get_issue(url):
         print('Unable to load issue %s' % url)
         return
     return issue
+
+
+def get_logo(org_name, size=0):
+    if size != 0:
+        image_url_short = 'http://github.com/%s.png?size=%s' % (org_name, size)
+    else:
+        image_url_short = 'http://github.com/%s.png' % (org_name)
+
+    # Follow the redirect to the page containing the image and
+    # store it in the response variable
```

Comment:
and this comment was inaccurate, as there is no response variable here.
(but you need a response variable :P)

anyway, kill the comment; it is useless, and useless comments are usually incorrect comments.
Nicely written code is better than comments.